src/components/TopicsList.jsx

```diff
@@ -0,0 +1,34 @@
+import { useEffect, useState } from "react";
+import { getTopics } from "../utils/api";
+import TopicCard from "./TopicCard";
+
+const Topics = () => {
+    const [topics, setTopics] = useState([])
+
+    const [isLoading, setIsLoading] = useState(true)
+    
+    useEffect(() => {
+        getTopics().then((res) => {
+            setTopics(res.data.topics)
+            setIsLoading(false)
+        });
+    }, []);
+
+
+    if(isLoading) {
+        return <p>Loading...</p>
+    }
+
+    return (
+        <section className="topics">
+            <h2>All Topics</h2>
+            {topics.map((topic) => {
```

Comment:
This works great but I would make it a bit more obvious these topics are clickable 👓